interface-definitions/container.xml.in

```diff
@@ -104,11 +104,38 @@
               </leafNode>
             </children>
           </tagNode>
+          <leafNode name="entrypoint">
+            <properties>
+              <help>Override the default ENTRYPOINT from the image</help>
+              <constraint>
+                <regex>[ !#-%&amp;(-~]+</regex>
+              </constraint>
+              <constraintErrorMessage>Entrypoint must be ascii characters, use &amp;quot; and &amp;apos for double and single quotes respectively</constraintErrorMessage>
```

Comment:
We should only support `&amp;quot;` as we do in other nodes. No need for both.

Reply:
It is needed for the json-formatted argument, and I think that when using raw command, people may need to use both types of quotes

https://github.com/Zen3515/vyos-1x/blob/53aebddb4ca54b0cc4a296d6cc4c4d960c5f1d73/src/conf_mode/container.py#L284-L286

https://docs.podman.io/en/latest/markdown/podman-run.1.html#entrypoint-command-command-arg1
 
 So, I thought why not include it.
 Should I remove it anyway?

Reply:
I think supporting both quotes here is logical.